test/e2e/operator_expectations.go

```diff
@@ -13,7 +14,7 @@ import (
 	cvorm "github.com/openshift/cluster-version-operator/lib/resourcemerge"
 	kappsapi "k8s.io/api/apps/v1"
 	batchv1 "k8s.io/api/batch/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
maybe corev1 here?  Did you mean to add this change, seems a little out of place for this PR.

Reply:
Automatically generated

Reply:
Yeah, this is from running `goimports`.